pkg/cmd/synonyms/shared/flags_to_synonym.go

```diff
@@ -0,0 +1,137 @@
+package shared
+
+import (
+	"fmt"
+
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+	"github.com/algolia/cli/pkg/config"
+	"github.com/algolia/cli/pkg/iostreams"
+)
+
+type SynonymFlags struct {
+	SynonymID           string
+	SynonymInput        string
+	SynonymWord         string
+	SynonymPlaceholder  string
+	SynonymType         SynonymType
+	Synonyms            []string
+	SynonymCorrections  []string
+	SynonymReplacements []string
+}
+
+type SaveOptions struct {
```

Comment:
I tried to do that at first but couldn't make it work because of the import cycle error... Should I create a separate file only for this type declaration?

Reply:
Looks like the `SaveOptions` is used in the `GetSuccessMessage`, but it could be avoided (just pass the `Indice` as an arg, and maybe add the `successIcon` inside the command function?